.github/workflows/ci_cron.yml

```
@@ -6,49 +6,50 @@ on:
     - cron: "0 6 * * 1"
```

```suggestion
    - cron: "0 0 * * 1"
```
move scheduled workflow to midnight